doc/topics/cloud/gce.rst

```diff
@@ -536,6 +546,42 @@ Specify the network name to view information about the network.
 
     salt-cloud -f show_network gce name=mynet
 
+Create subnetwork
+--------------
+New subnetworks require a name, region, and CIDR range.
+Optionally, 'description' can be provided to add an extra note to your subnetwork.
+New instances can be created and added to this subnetwork by setting the subnetwork name during create. It is
+not possible to add/remove existing instances to a subnetwork.
+
+.. code-block:: bash
+
+    salt-cloud -f create_subnetwork gce name=mynet network=mynet region=us-central1 cidr=10.0.10.0/24
+    salt-cloud -f create_subnetwork gce name=mynet network=mynet region=us-central1 cidr=10.10.10.0/24 description=some info about my subnet.
+
+.. versionadded:: Nitrogen
+
+Destroy subnetwork
+---------------
+Destroy a subnetwork by specifying the name and region. Make sure that there are no
+instances associated with the network prior to deleting it or you'll have
+a bad day.
```

Comment:
I'm sure it would be a bad day but it would be better here to know exactly what would happen. :] Will the instances just stop working? Will they be deleted? I think we need to explain this failure case a little better here.